src/components/projects/MemberList.jsx

```diff
@@ -0,0 +1,119 @@
+import DeleteIcon from "@material-ui/icons/Delete";
+import IconButton from "@material-ui/core/IconButton";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction";
+import ListItemText from "@material-ui/core/ListItemText";
+import MenuItem from "@material-ui/core/MenuItem";
+import TextField from "@material-ui/core/TextField";
```

Comment:
you can use the group wey to import material-ui components something like this:
import {
  Button,
  Paper,
  TableCell,
  TableContainer,
  TableRow,
  Typography,
} from "@material-ui/core";

Reply:
done